lib/bcdice/game_system/SwordWorld.rb

```diff
@@ -301,6 +309,7 @@ def rollDice(_command)
       # @param rateResults  [Array<String>]
       # @param dice_total [Integer]
       # @param round [Integer]
+      # @return [Array<String, Boolean>] output, critical, fumble
```

Comment:
```suggestion
      # @return [Array(String, Boolean, Boolean)] output, critical, fumble
```

YARDではArrayをTupleのように使う場合このように書きます。